config/src/trusted_peers.rs

```diff
@@ -44,13 +45,19 @@ pub struct NetworkPrivateKeys {
     pub network_identity_private_key: X25519StaticPrivateKey,
 }
 
-#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+#[derive(Clone, Default, Serialize, Deserialize)]
 pub struct NetworkPeersConfig {
     #[serde(flatten)]
     #[serde(serialize_with = "serialize_ordered_map")]
     pub peers: HashMap<String, NetworkPeerInfo>,
 }
 
+impl fmt::Debug for NetworkPeersConfig {
```

Comment:
why not impl Display and change the caller to be {}?

Reply:
Because it prints it as part of bigger struct